homeassistant/components/sungrow/const.py

```diff
@@ -0,0 +1,181 @@
+        name="voltage AC3",
+        native_unit_of_measurement=ELECTRIC_POTENTIAL_VOLT,
+        device_class=SensorDeviceClass.VOLTAGE,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    # '5022 - Phase A current'
+    SungrowSensorEntityDescription(
+        key="5022 - Phase A current",
+        name="current AC1",
+        native_unit_of_measurement=ELECTRIC_CURRENT_AMPERE,
+        device_class=SensorDeviceClass.CURRENT,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    # '5023 - Phase B current'
+    SungrowSensorEntityDescription(
+        key="5023 - Phase B current",
+        name="current AC2",
+        native_unit_of_measurement=ELECTRIC_CURRENT_AMPERE,
+        device_class=SensorDeviceClass.CURRENT,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    # '5024 - Phase C current'
+    SungrowSensorEntityDescription(
+        key="5024 - Phase C current",
+        name="current AC3",
+        native_unit_of_measurement=ELECTRIC_CURRENT_AMPERE,
+        device_class=SensorDeviceClass.CURRENT,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    # '5003 - Daily power yields'
+    SungrowSensorEntityDescription(
+        key="5003 - Daily power yields",
+        name="yield day",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        device_class=SensorDeviceClass.ENERGY,
+        value=lambda value: round(value, 3),
+    ),
+    # '5144 - Total power yields'
+    SungrowSensorEntityDescription(
+        key="5144 - Total power yields",
+        name="yield total",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        device_class=SensorDeviceClass.ENERGY,
+        state_class=SensorStateClass.TOTAL,
+        value=lambda value: round(value, 1),
+    ),
+    # '5113 - Daily running time'
+    SungrowSensorEntityDescription(
+        key="5113 - Daily running time",
+        name="running time today",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=TIME_MINUTES,
+        device_class=SensorDeviceClass.DURATION,
+        state_class=SensorStateClass.TOTAL,
+    ),
+    # '5006 - Total running time'
+    SungrowSensorEntityDescription(
+        key="5006 - Total running time",
+        name="running time total",
+        icon="mdi-clock-start",
+        native_unit_of_measurement=TIME_HOURS,
+        device_class=SensorDeviceClass.DURATION,
+        state_class=SensorStateClass.TOTAL,
+        value=lambda value: round(value, 1),
+    ),
+    # '5001 - Nominal active power'
+    SungrowSensorEntityDescription(
+        key="5001 - Nominal active power",
+        name="installed peak power",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=POWER_KILO_WATT,
+        device_class=SensorDeviceClass.POWER,
+    ),
+    # '5017 - Total DC power' - '5031 - Total active power'
+    SungrowSensorEntityDescription(
+        key="alternator_loss",
```

Comment:
Is this correct? Doesn't seem to fit with the comment or the format of the other keys

Reply:
alternator loss is calculated from the two values mentioned in the comment. This is done when the sensor values are updated.

Reply:
ok, yes i see now. :)